Karma-Khameleons/bangazon-website
# Ls categoryrefactor

## Description
Add description about what this pull request covers.
This PR covers refactoring both the index and category views to show the products in said categories in order of last ten and last five added. 

## Number of Fixes
List of fixes included in this PR.
No fixes, all new code. 

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#52, #53 

## Expected Behavior
Describe the expected behavior.
Index page should show headers for the categories and links for last five products added to the category. Clicking the link should take you to product detail for specific product. On the categories page, the behavior should be similar except there are 10 products displayed (when we have ten) and the category headers are links as well. 

## Steps to Test Solution
List steps to test your solution
`Git fetch` this branch
`Python manage,py runserver`
Navigate through the website and check for described behavior on the two pages.
